src/test/java/com/wiley/autotest/framework/pages/TestDomElementPage.java

```diff
@@ -20,6 +22,13 @@ public TestDomElementPage checkDomElementBecomeVisible() {
         return this;
     }
 
+    public TestDomElementPage checkDomElementAppearAfterTimeout() {
```

Comment:
I don't see any timeouts and checks for timeout in method.

Reply:
the "logic" is in the HTML page... (an element is attached to dom after 6 seconds). Will add a comment to the method - unless you have some suggestions on how to better name it?